inference-engine/ie_bridges/python/sample/hello_reshape_ssd/hello_reshape_ssd.py

```diff
@@ -0,0 +1,210 @@
+            net.input_info[input_key].precision = 'FP32'
+            if net.input_info[input_key].input_data.shape[1] != 3 and net.input_info[input_key].input_data.shape[1] != 6 or \
+                net.input_info[input_key].input_data.shape[0] != 1:
+                log.error('Invalid input info. Should be 3 or 6 values length.')
+
+    data = {}
+    data[input_name] = images
+
+    if input_info_name != "":
+        infos = np.ndarray(shape=(n, c), dtype=float)
+        for i in range(n):
+            infos[i, 0] = h
+            infos[i, 1] = w
+            infos[i, 2] = 1.0
+        data[input_info_name] = infos
+
+    # --------------------------- Prepare output blobs ----------------------------------------------------
+    log.info('Preparing output blobs')
+
+    output_name, output_info = "", net.outputs[next(iter(net.outputs.keys()))]
+    output_ops = {op.friendly_name : op for op in ops \
+                  if op.friendly_name in net.outputs and op.get_type_name() == "DetectionOutput"}
+    if len(output_ops) != 0:
+        output_name, output_info = output_ops.popitem()
+
+    if output_name == "":
+        log.error("Can't find a DetectionOutput layer in the topology")
+
+    output_dims = output_info.shape
+    if len(output_dims) != 4:
+        log.error("Incorrect output dimensions for SSD model")
+    max_proposal_count, object_size = output_dims[2], output_dims[3]
+
+    if object_size != 7:
+        log.error("Output item should have 7 as a last dimension")
+
+    output_info.precision = "FP32"
+    # -----------------------------------------------------------------------------------------------------
+
+    # --------------------------- Performing inference ----------------------------------------------------
+    log.info("Loading model to the device")
+    exec_net = ie.load_network(network=net, device_name=args.device)
+    log.info("Creating infer request and starting inference")
+    res = exec_net.infer(inputs=data)
+    # -----------------------------------------------------------------------------------------------------
+
+    # --------------------------- Read and postprocess output ---------------------------------------------
+    log.info("Processing output blobs")
+    res = res[out_blob]
+    boxes, classes = {}, {}
+    data = res[0][0]
+    for number, proposal in enumerate(data):
+        if proposal[2] > 0:
+            imid = np.int(proposal[0])
+            ih, iw = images_hw[imid]
+            label = np.int(proposal[1])
+            confidence = proposal[2]
+            xmin = np.int(iw * proposal[3])
+            ymin = np.int(ih * proposal[4])
+            xmax = np.int(iw * proposal[5])
+            ymax = np.int(ih * proposal[6])
+            print("[{},{}] element, prob = {:.6}    ({},{})-({},{}) batch id : {}" \
+                  .format(number, label, confidence, xmin, ymin, xmax, ymax, imid), end="")
+            if proposal[2] > 0.5:
+                print(" WILL BE PRINTED!")
```

Comment:
is it debug print? please remove or update text without capslock.

Reply:
Refactored output, fixed